### PR TITLE
Update uml

### PR DESCRIPTION
**Pull request by bryansendeavour**

### CODE REVIEW

File: docs/images/NewContactActivityDiagram.png
Comment (glozxi) [inline docs/images/NewContactActivityDiagram.png:1]: Should the end node be changed to dot inside circle notation?

**Review by glozxi — CHANGES_REQUESTED** [review]
Other than a nit, LGTM 😎

File: docs/images/NewContactActivityDiagram.png
Comment (bryansendeavour) [reply docs/images/NewContactActivityDiagram.png:1]: Great catch!

Comment (glozxi) [reply docs/images/NewContactActivityDiagram.png:1]: Remember to change the png too!

**Review by glozxi — APPROVED** [review]
LGTM 😄

**Pull request merged**